stanford-earth/matson
# EARTH-1019: Video headers above except for item at the top of the page.

# READY FOR REVIEW

# Summary
- Sets all video header text to above the video on mobile except for the very first hero video as long as it is the first item in the paragraph.

# Steps to Test

1. Check out this branch
2. Run `npm install`
3. Run `grunt sass`
4. Clear caches
5. Navigate to the home page
6. Resize browser window to see header go below video
7. Edit home page and move the top video down a couple of paragraphs
8. Resize browser window to see header go above video.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
